device-identifier/shbdeviceidentifier/app.py

```diff
@@ -197,55 +197,17 @@ def query(ctx, data_base, statement_name):
 
 
 @app.command("train")
-@click.argument("model-selector", nargs=1, type=click.STRING)
+@click.argument("model-name", nargs=1, type=click.STRING)
 @click.argument("training-data-path", nargs=1, type=click.STRING)
 @click.argument("training-labels-path", nargs=1, type=click.STRING)
+@click.argument("use-database", nargs=1, type=click.BOOL, default=False)
```

Comment:
Would love to have this. I tried implementing `use-database` as an option, but could not get it to work...

Reply:
Could you try if you can get it to work?

A valid CLI call for testing would be `shbdeviceidentifier --debug train "" "" "" True "2022-08-01 11:38:02" "2022-08-01 11:41:02"`. Make sure to read our pcap into the db first.

Reply:
I'll take a look today.

Reply:
I completely changed the implementation to make it possible to use the model as described above. I did not plan on changing everything, that's why I committed to this branch, which looks dumb in hindsight. I'd cherry pick into a new branch if necessary. Anyway, can now be used like this:

```bash
shbdeviceidentifier set-measurement "myData"
shbdeviceidentifier read /path/to/trainingdata.pcap
shbdeviceidentifier read-labels /path/to/traininglabels.json
shbdeviceidentifier train RandomForest
```

Alternatively each command accepts the measurment flag
```bash
shbdeviceidentifier read --measurement "myData" /path/to/trainingdata.pcap
[...]
```

Reply:
The new commands look way better than before. However, I would like to keep ability to set timestamps for splitting the data into train and test sets. With the increasing feature set, also in the GUI, it might might be a good time to discuss the overall workflow of reading data, training a model, and testing/applying the model to unseen data from a user's point of view.

How about we discuss this during the next meeting?

Reply:
I updated the code a bit. Should work now, together with the other PRs.